src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -57,7 +57,10 @@ public function validateActiveUrl($attribute, $value)
 
         if ($url = parse_url($value, PHP_URL_HOST)) {
             try {
-                return count(dns_get_record($url, DNS_A | DNS_AAAA)) > 0;
+                return (bool) dns_get_record(
+                    idn_to_ascii($url, IDNA_DEFAULT, INTL_IDNA_VARIANT_UTS46),
```

Comment:
The `idn_to_ascii` function will not exist on most systems.

Reply:
Please add the dependency`"polyfill-intl-idn": "^1.11"` to both the main `composer.json` and also the validation component's `composer.json`.

Reply:
Thanks for your review, I added the dependency to both `composer.json`s.